lua/astrotheme/lib/util.lua

```diff
@@ -1,5 +1,7 @@
 local M = {}
 
+function M.set_contains(set, key) return set[key] ~= nil end
+
```

Comment:
```suggestion
```

this exists with `vim.tbl_contains`. Also this is not semantically correct because this is operating on dictionary like tables and not set like lists.

Reply:
Thank you, I guess I better Google harder lol.
..Although it seems to work fine. :thinking:

Reply:
I see now, thanks!!